_rules/element-lang-matches-common-language-off6ek.md

```diff
@@ -0,0 +1,323 @@
+---
+id: off6ek
+name: HTML element language subtag matches language
+rule_type: atomic
+description: |
+  This rule checks that the primary language subtag of an element matches its language
+accessibility_requirements:
+  wcag20:3.1.2: # Language of Parts (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: further testing needed
+  wcag-technique:H58: # Using language attributes to identify changes in the human language
+    forConformance: false
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+input_aspects:
+  - DOM tree
+  - Accessibility tree
+  - CSS Styling
+  - Language
+acknowledgments:
+  authors:
+    - Jean-Yves Moyen
+htmlHintIgnore:
+  # https://www.npmjs.com/package/htmlhint
+  # using aria-labelledby instead
+  - 'alt-require'
+---
+
+## Applicability
+
+This rule applies to any HTML element for which all the following are true:
```

Comment:
I know this is a bit redundant, but I think the rule is easier to understand if we do this.

```suggestion
This rule applies to any HTML element with a `lang` attribute for which all the following are true:
```